.github/dependabot.yml

```diff
@@ -13,7 +13,7 @@ updates:
 
   - package-ecosystem: "github-actions"
     directories:
-      - "**/*"
+      - "!(requirements)/**/*"
```

Comment:
I am not even sure if the `!` negate is supported by the glob implemented by GitHub. We can merge and manually trigger dependabot to see how it behaves. Right now, it is broken with the status in main.

Reply:
Sounds good!